db/migrate_cards/data/cards/company-Xself-Xstructure

```diff
@@ -1,9 +1,9 @@
-<p>{{+description}}</p>

```

Comment:
I don't know why these non-changes are showing in all these renamed (because of the Windows thing) files